tests/helpers.py

```diff
@@ -1907,3 +1907,67 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def validate_pods_are_running_and_not_restarted(
```

Comment:
I don't think that this function should be in helpers as it seems specific for one test case.
 Also why provided arguments start with previous? I don't understand that from the function docstring.

Reply:
Agreed, removed from helpers.py